packages/genui/playground/src/pages/bench/PhaseTwoReportPage.css

```
@@ -0,0 +1,2191 @@
+    in srgb,
+    var(--phase-report-a2ui) 10%,
+    var(--phase-report-paper)
+  );
+  --phase-report-openui: #c65c2b;
+  --phase-report-openui-soft: color-mix(
+    in srgb,
+    var(--phase-report-openui) 10%,
+    var(--phase-report-paper)
+  );
+  --phase-report-data:
+    var(--geist-mono), "SFMono-Regular", Consolas, "Liberation Mono", monospace;
```

Stylelint (17.14.1)

[error] Expected "Consolas" to be "consolas" (value-keyword-case)

(value-keyword-case)